lib/model/store.dart

```diff
@@ -345,6 +345,64 @@ abstract class GlobalStore extends ChangeNotifier {
     ));
   }
 
+  /// Attempt to refresh the realm metadata for each account.
+  ///
+  /// Fetches server settings for each account and then calls [updateRealmData]
+  /// to update the realm metadata (name and icon).
+  ///
+  /// The updation for some accounts may be skipped if:
+  ///  - The [PerAccountStore] for the account is already loaded or is loading.
+  ///  - The version of corresponding realm server is unsupported.
```

Comment:
I can't quite parse this last line. Does this cover what you mean?

```suggestion
  ///  - The Zulip Server version of the realm's server is unsupported.
```